baselines/aws/aws_v3_v5_migration/variables.tf

```diff
@@ -32,3 +32,12 @@ variable "turbot_cloudtrails_prefix" {
   description = "Prefix for the Turbot-managed CloudTrail. Defaults value out of the box from Turbot V5 to 'turbot-'.  Applies to buckets and cloudtrails."
   type = string
 }
+
+variable "aws_approved_regions" {
+  description = "YAML list of approved AWS regions."
+  type        = string
+  default     = <<EOT
+  - us-east-1
+  - us-east-2
```

Comment:
You should remove this, the default region is very custom. If you remove this, then terraform apply will ask the question rather than silently do the task.